angular-hub/src/app/components/event-card.component.ts

```diff
@@ -15,14 +16,20 @@ import {DatePipe, NgIf, NgOptimizedImage} from "@angular/common";
               <span class="font-bold text-[#BF25B9]" itemprop="date">{{event.attributes.date | date}}</span>
               <h3 [attr.id]="event.attributes.title" class="text-xl font-bold" itemprop="title">{{event.attributes.title}}</h3>
               <span class="text-gray-500" itemprop="location">{{event.attributes.location}}</span>
+              <div class="flex gap-2">
+                  @for (tag of event.attributes.tags; track tag) {
```

Comment:
please use *ngFor: I'm waiting for broader support with prettier to use new control flow (having plans to use @empty too with the project)